pkg/inspector/config.go

```diff
@@ -0,0 +1,62 @@
+package inspector
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+)
+
+const ConfigFilename = "kusari.yaml"
+
+type Config struct {
+	GitHubActionVersionPinningCheckEnabled bool
+	ContainerVersionPinningCheckEnabled    bool
+}
+
+type configYAML struct {
+	GitHubActionVersionPinningCheckEnabled *bool `yaml:"github_action_version_pinning_check_enabled,omitempty"`
+	ContainerVersionPinningCheckEnabled    *bool `yaml:"container_version_pinning_check_enabled,omitempty"`
+}
```

Comment:
If we want to keep the config defs in this repo, and import elsewhere, put them in `api/`

Reply:
Fixed in https://github.com/kusaridev/kusari-cli/pull/45/commits/8fde287546e30ea9f72eb2b8fbeee37b6af9ce6c